features/webgl-context-colorspace.yml

```diff
@@ -0,0 +1,8 @@
+name: WebGL Color Space
```

Comment:
I'm open to splitting this into two features, one has baseline low, the other is only supported in Chrome/Edge.